scilpy/utils/streamlines.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python
```

Comment:
You can remove shebang if you want, this is not a script

Reply:
Will do a pass tomorrow with the newbies, deal ?